style.css

```diff
@@ -0,0 +1,137 @@
+#nav {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 50px;
+}
+
+#nav a {
+  text-decoration: none;
+  font-family: Poppins, sans-serif;
+  font-weight: bold;
+  font-size: 30px;
+  color: #6070ff;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  margin-left: 24px;
+}
+
+#nav a:hover {
+  text-decoration: underline;
+  -webkit-text-decoration-color: #23a3fa;
+  text-decoration-color: #23a3fa;
+  text-decoration-thickness: 2px;
+}
+
+#nav nav {
+  width: 45px;
+  height: 30px;
+  display: flex;
+  align-items: center;
+}
+
+section {
+  background-image:
```

Comment:
- [ ] [OPTIONAL] It's possible to download and use the background image at once. see a demo below.

https://www.loom.com/share/371616c731d14be1abfdda6805264807